src/components/Single-review.jsx

```diff
@@ -62,7 +75,30 @@ export const SingleReview = () => {
 				</ul>
 			)}
 			<li className='post-comment'></li>
-			<li className='comments'></li>
+			{isComments ? (
+				<li className='comments'>
+					<ul className='comments-list'>
+						{comments.map((comment) => {
+							return (
+								<li key={comment.comment_id} className='comment'>
+									<section className='comment-user-box'>
+										<span className='material-symbols-outlined'>
+											account_circle
+										</span>
+										<p>{comment.author}</p>
+										<p className='comment-created'> on {comment.created_at}</p>
```

Comment:
I would eventually look at having a util function format this created_at timestamp into a more readable format for front-end users. It can then be used across multiple components